src/plasoscaffolder/bll/services/base_sqlite_generator.py

```diff
@@ -1,39 +1,43 @@
 # -*- coding: utf-8 -*-
+"""Baseclass for Generator for Sqlite"""
 import abc
 import os
 
-from plasoscaffolder.bll.mappings.base_init_mapping import BaseInitMapper
-from plasoscaffolder.bll.mappings.base_mapping_helper import BaseMappingHelper
-from plasoscaffolder.bll.services.base_sqlite_plugin_path_helper import \
-  BaseSQLitePluginPathHelper
-from plasoscaffolder.bll.services.sqlite_plugin_helper import \
-  BaseSQLitePluginHelper
-from plasoscaffolder.common.base_file_handler import BaseFileHandler
-from plasoscaffolder.common.base_output_handler import BaseOutputHandler
-from plasoscaffolder.common.file_handler import FileHandler
+from plasoscaffolder.bll.mappings import base_formatter_mapping
+from plasoscaffolder.bll.mappings import base_init_mapping
+from plasoscaffolder.bll.mappings import base_mapping_helper
+from plasoscaffolder.bll.mappings import base_parser_mapping
+from plasoscaffolder.common import base_file_handler
 
 
 class BaseSQLiteGenerator(object):
   """Class representing the base class for the base sqlite generator."""
   __metaclass__ = abc.ABCMeta
 
   @abc.abstractmethod
-  def generate_sqlite_plugin(self, template_path: str,
-      fileHandler: BaseFileHandler, init_mapper: BaseInitMapper,
-      mappingHelper: BaseMappingHelper):
+  def GenerateSQLitePlugin(
+      self,
+      template_path: str,
+      fileHandler: base_file_handler.BaseFileHandler,
+      init_mapper: base_init_mapping.BaseInitMapper,
+      parser_mapper: base_parser_mapping.BaseParserMapper,
+      formatter_mapper: base_formatter_mapping.BaseFormatterMapper,
+      mappingHelper: base_mapping_helper.BaseMappingHelper):
     """Generate the whole sqlite plugin.
```

Comment:
Sorry for the confusion this is my short hand for:
Please change "Generate the whole sqlite plugin." into "Generate a SQLite plugin."